src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1459,6 +1459,9 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
 
     aad_profile = None
     if any([aad_client_app_id, aad_server_app_id, aad_server_app_secret, aad_tenant_id]):
+        if aad_tenant_id is None:
+            _, _, aad_tenant_id = profile.get_login_credentials()
```

Comment:
`profile` isn't defined yet in this context--we need to add this line just before it:
`profile = Profile(cli_ctx=cmd.cli_ctx)`

Reply:
If the aad_tenant_id will never ever be a different from the current account, then this argument should not be exposed, I would suggest deprecate it.

Reply:
`aad_tenant_id` can be different from customer's subscription tenant. This can happen where customer sets up a separate AAD tenant for AAD auth than the tenant where their subscription lives.